staging/src/k8s.io/kubectl/pkg/cmd/util/editor/editoptions.go

```diff
@@ -437,6 +454,49 @@ func (o *EditOptions) Run() error {
 	}
 }
 
+func (o *EditOptions) extractManagedFields(obj runtime.Object) error {
```

Comment:
A test for extract / clear / restore would be good?

Reply:
Done